src/components/Settings/SettingsTab.tsx

```diff
@@ -123,6 +132,14 @@ export const SettingsTab = ({
         disabled={lockDisabled}
         onChange={(e, data) => setRoomLock(data.checked)}
       />
+      <SettingRow
```

Comment:
think we want to make this a permanent/admin setting like passwords and custom names